charts/spire/charts/spire-server/templates/controller-manager-webhook.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: {{ include "spire-controller-manager.fullname" . }}-webhook
+  name: {{ .Release.Namespace }}-{{ include "spire-controller-manager.fullname" . }}-webhook
```

Comment:
Are you planning to deploy the parallel controller managers in different namespaces?

Reply:
Yes.

For one cluster, I'm currently planning on 3.
spire-root - root ca for all other spires.
spire-server - regular examples/production deployment for the k8s cluster its on, but ca is from spire-root
spire-external - exported out of the cluster for non k8s workloads. ca is from spire-root.

Each instance would be hardened different ways for their particular use cases.

Reply:
the release name would be the same for all 3?

Reply:
Yeah, that's how we typically name things. For singletons in their own namespaces, release name = chart name. The namespace differentiates things.

We've got other places in the main branch where we have added the .Release.Namespace into global objects to prevent naming conflict. This one was just missed as we didn't support multiple controller-managers til now.

Reply:
For RBAC reasons I get the splitting of namespaces? But what if someone decides to put all in the same namespace with different release names? Should we support that?

Reply:
That should work as well I think? The full name of the release is included in the name, so should be unique when multiple are in the same namespace.